.github/workflows/validate_datasets.yml

```
@@ -60,13 +61,37 @@ jobs:
         bash -c "npm install -g bids-validator-*.tgz"
         popd
 
+    - uses: denoland/setup-deno@v1.1.2
+      if: "matrix.bids-validator == 'master-deno'"
+      with:
+        deno-version: v1.x
+
+    - name: Install BIDS validator (master deno build)
+      if: "matrix.bids-validator == 'master-deno'"
+      run: |
+        set -x
+        pushd ..
+        git clone --depth 1 https://github.com/bids-standard/bids-validator
+        # For debugging purposes
+        echo "PATH=$PATH"
+        VALIDATOR=/usr/bin/bids-validator
+        sudo bash -c 'echo -e "#!/bin/sh\n""$PWD/bids-validator/bids-validator/bids-validator-deno \"\$@\"" > /usr/bin/bids-validator'
+        cat $VALIDATOR
+        sudo chmod a+rx $VALIDATOR
+        which -a bids-validator
+        bids-validator --version
+        bids-validator --help
+        popd
```

So I just created a working validator in my `$HOME/.local/bin`:

```typescript
#!/bin/env -S deno run --allow-read --allow-write --allow-env --allow-net --allow-run
import 'https://github.com/bids-standard/bids-validator/raw/master/bids-validator/src/bids-validator.ts'
```

So I would suggest seeing if we can do the same, as we should always have write access to $HOME:

```suggestion
        LOCAL_BIN=$HOME/.local/bin
        VALIDATOR=$LOCAL_BIN/bids-validator
        echo $VALIDATOR  # debug
        mkdir -p $LOCAL_BIN
        export PATH="$LOCAL_BIN:$PATH"
        echo PATH="$PATH" | tee -a $GITHUB_ENV
        echo -e '#!/bin/env'" -S deno run --allow-read --allow-write --allow-env --allow-net --allow-run\nimport 'https://github.com/bids-standard/bids-validator/raw/master/bids-validator/src/bids-validator.ts'" > $VALIDATOR
        chmod +x $VALIDATOR
        # For debugging purposes
        cat $VALIDATOR
        which -a bids-validator
        bids-validator --version
        bids-validator --help
```